### PR TITLE
git51366: Add vCenter UI permissions

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.8+

Issue: https://github.com/openshift/openshift-docs/issues/51366

Link to docs preview: https://52846--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-installer-infra-requirements_installing-vsphere-installer-provisioned

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 1]
```diff
@@ -31,14 +31,14 @@ If you cannot use an account with global administrative privileges, you must cre
 
 An additional role is required if the installation program is to create a vSphere virtual machine folder.
 
-.Roles and privileges required for installation
+.Roles and privileges required for installation in vSphere UI
```
Comment (sgaoshang) [inline modules/installation-vsphere-installer-infra-requirements.adoc:34]: `in vSphere API` is more precise IMO.

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 2]
```diff
@@ -31,14 +31,14 @@ If you cannot use an account with global administrative privileges, you must cre
...
 [%collapsible]
 ====
 [cols="3a,3a,3a",options="header"]
 |===
 |vSphere object for role
 |When required
-|Required privileges
+|Required privileges in vSphere UI
```
Comment (sgaoshang) [inline modules/installation-vsphere-installer-infra-requirements.adoc:41]: in vSphere API is more precise IMO.

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 3]
```diff
@@ -161,6 +161,130 @@ An additional role is required if the installation program is to create a vSpher
...
 ====
 
+.Roles and privileges required for installation in vCenter UI
+[%collapsible]
+====
+[cols="3a,3a,3a",options="header"]
+|===
+|vSphere object for role
+|When required
+|Required privileges in vCenter UI
```
Comment (sgaoshang) [inline modules/installation-vsphere-installer-infra-requirements.adoc:171]: Change to "Required privileges in vCenter GUI"?

File: modules/installation-vsphere-installer-infra-requirements.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/installation-vsphere-installer-infra-requirements.adoc]: Addressed

File: modules/installation-vsphere-installer-infra-requirements.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/installation-vsphere-installer-infra-requirements.adoc]: Addressed

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 4]
```diff
@@ -161,6 +161,130 @@ An additional role is required if the installation program is to create a vSpher
...
+[%hardbreaks]
+`Host.Configuration."Storage partition configuration"`
+`Resource."Assign virtual machine to resource pool"`
+`VApp."Assign resource pool"`
+`VApp.Import`
+`"Virtual machine"."Change Configuration"."Add new disk"`
+
+|vSphere vCenter Resource Pool
+|If an existing resource pool is provided
+|None
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/installation-vsphere-installer-infra-requirements.adoc]: Addressed

File: modules/installation-vsphere-installer-infra-requirements.adoc (hunk 3 above)
Comment (xenolinux) [reply modules/installation-vsphere-installer-infra-requirements.adoc]: Addressed

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 5]
```diff
@@ -161,6 +161,136 @@ An additional role is required if the installation program is to create a vSpher
 |===
 ====
 
+.Roles and privileges required for installation in vCenter UI
```
Comment (sgaoshang) [inline modules/installation-vsphere-installer-infra-requirements.adoc:164]: I missed here `in vCenter UI` to `in vCenter GUI`

Comment (xenolinux) [reply modules/installation-vsphere-installer-infra-requirements.adoc]: Addressed

**Review by JoeAldinger — COMMENTED** [review]
One small comment. 
/lgtm

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 6]
```diff
@@ -161,6 +161,136 @@ An additional role is required if the installation program is to create a vSpher
...
+
+|vSphere Port Group
+|Always
+|`Network."Assign network"`
+
+|Virtual Machine Folder
+|Always
+|
+[%hardbreaks]
+``"vSphere Tagging"."Assign or Unassign vSphere Tag on Object"`
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/installation-vsphere-installer-infra-requirements.adoc]: ah right good catch addressed

**Pull request merged**